resolvers/template.js

```diff
@@ -1,5 +1,2 @@
-const authCheck = require('./functions/authCheck');
-
-module.exports = async (parent, args, context) => {
-  authCheck(context);
-};
+/* eslint-disable no-unused-vars */
+module.exports = async (parent, args, context) => {};
```

Comment:
Can you explain this function?

Reply:
I am not exactly sure. It was present earlier as an empty resolver with only the `authCheck`